lib/utils/local_storage.dart

```diff
@@ -41,7 +41,8 @@ enum LocalStorage {
   themeMode,
   navBarSelection,
   adminWatchLive,
-  adminStream;
+  adminStream,
+  notify;
```

Comment:
I think it makes a lot of sense for stuff like display & notification settings to be saved in local storage.

With that in mind, I wonder if we could get rid of the class member in **user.dart** and make it work just using the `LocalStorage.notify()` value 🤔